demos/rgb/rgb.js

```diff
@@ -38,9 +38,9 @@
       port.send(view);
     };
 
-    redSlider.addEventListener('change', onUpdate);
-    greenSlider.addEventListener('change', onUpdate);
-    blueSlider.addEventListener('change', onUpdate);
+    redSlider.addEventListener('input', onUpdate);
+    greenSlider.addEventListener('input', onUpdate);
+    blueSlider.addEventListener('input', onUpdate);
```

Comment:
I've found that my Arduino isn't fast enough to respond to requests quickly enough to handle the barrage of events generated by listening for `oninput`. It would be an interesting exercise to see if the sketch could be improved to handle this better. Removing the printed response to each command would likely help but it would be more interesting to modify the WebUSB library itself into a better example of of a command/response protocol though looking at the Arduino's USB library it may not be practical to improve latency much. Control transfers may be an interesting path to investigate.